### PR TITLE
Add gogo compilers to protoc-go

### PR DESCRIPTION
**Pull request by kuujo**
This PR adds the gogo Protobuf compiler to the protoc-go image. This allows us to generate faster Protobuf serializers and use special features for Golang code generation.

### CODE REVIEW

File: build/protoc-go/Dockerfile [hunk 1]
```diff
@@ -3,6 +3,10 @@ FROM golang:1.12
 RUN apt-get update && \
     apt-get install -y protobuf-compiler && \
     go get -u github.com/golang/protobuf/protoc-gen-go && \
+    go get -u github.com/gogo/protobuf/protoc-gen-gofast && \
+    go get -u github.com/gogo/protobuf/protoc-gen-gogofast && \
+    go get -u github.com/gogo/protobuf/protoc-gen-gogofaster && \
```
Comment (tomikazi) [inline build/protoc-go/Dockerfile]: I'm curious why you decided not to use protoc-gen-gogogofastest though.

Comment (kuujo) [reply build/protoc-go/Dockerfile]: Too fast

Comment (kuujo) [reply build/protoc-go/Dockerfile]: I didn't want to overheat my laptop

Comment (SeanCondon) [reply build/protoc-go/Dockerfile]: Jordan. Do you want to tackle issue #560 at the same time in this PR and upgrade to 3.9.0?

Comment (kuujo) [reply build/protoc-go/Dockerfile]: Sure sounds good. I have another small change to make to this one anyways.

**Comment by kuujo** [comment]
Should be good now.

**Comment by kuujo** [comment]
I'll push new `stable` and `latest` tags.

**Pull request merged**